### ISSUE TITLE
DCAT parsing issue on carriage returns

### ISSUE THREAD

**Issue by stijnvanhoey** [inbo/data-publication#149]
After the adaptations on the IPT, we still have issues on the carriage returns of the description fields. The reason is the requirement to use the 'next paragraph' options whenever a new paragraph (basically any `enter`) is required. 

This results in parsing issues with the dcat feed towards the open data Vlaanderen (and Belgium), resulting in not appearing in the listing. 

The [dcat feed output with the errors is ](https://github.com/inbo/data-publication/files/1406461/dcat.inbo.orig.txt) provides issues in the data sets PINK (amphibia), Dung beetles and Alien macro-invertebrates. 

We can check the well-functioning of the dcat feed with http://id.vlaanderen.be/vodap_validator
@DimEvil @milotictanja @peterdesmet @LienReyserhove 
Whenever we add new datasets, make sure to do a quick check on the validator by pasting the URL of the INBO dcat-feed (http://data.inbo.be/ipt/dcat) into the validator.

**Comment by stijnvanhoey** [comment]
I adapted the metadata for the data sets with the paragraph-issue and we do not have any parsing issues anymore. The result of the vodap_validator is: http://id.vlaanderen.be/vodap_validatorresults/2017-10-23T11:14:21Z/

The parsing issues are solved, but we still have some other errors in the report:

1. Catalog - error - dct:issued should be a literal typed as date or dateTime.
1. Catalog - error - dct:modified should be a literal typed as date or dateTime.

and some warnings:

1. License Document - warning - dct:LicenseDocument does not exist.
1. Agent - warning - dct:type is a recommended property for Agent.

@peterdesmet we should probably tackle these on the IPT implementation level?

**Comment by peterdesmet** [comment]
Yes, we should tackle these on an IPT implementation level. Maybe interesting to do as a 1 day project for a Java dev... or we could try ourselves.
